srcCxx/shape_main.cxx

```diff
@@ -1033,8 +1046,11 @@ class ShapeApplication {
 
                 if (retval == RETCODE_OK) {
                     auto n_samples = samples.length();
+                    logger.log_message("Read " + std::to_string(samples.length())
+                            + " sample(s), printing them...", Verbosity::DEBUG);
                     for (decltype(n_samples) i = 0; i < n_samples; i++)  {
-
+                        logger.log_message("Processing sample " + std::to_string(i),
+                                Verbosity::DEBUG);
```

Comment:
I would put this message just before the if in line 1065, and also log the value of `sample_info->valid_data`

Reply:
I thought about putting this function before the if, but in the end, nothing changes. I haven't printed the `valid_data` because in case that it is true, it will print the message always. Ideally we could print the sample_info standard fields, but I haven't put it in this PR.